cryk-ui/src/pages/CryptoInformation.tsx

```diff
@@ -159,6 +175,14 @@ export default function CryptoInformation(props: any) {
         context: { clientName: "newsGraphqlEndpoint" }
     });
 
+    const refetchInput2: RefetchInput<CryptocurrencyInput> = {
```

Comment:
Let's find a better name for `refetchInput` and `refetchInput2`. Maybe something like `getPaginatedNewsRefetchInput` and `getCryptoByIdRefetchInput`